### PR TITLE
Added ios support

### PR DESCRIPTION
**Pull request by BEARlogin**
Hi @awazgyawali ! Please check my PR with iOS support

### CODE REVIEW

File: ios/Classes/SwiftMultipartRequestPlugin.swift [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                    print(percents);
+                    SwiftMultipartRequestPlugin.channel?.invokeMethod("progress", arguments: String(percents))
+                       })
+            case .success(request: let upload, streamingFromDisk: true, streamFileURL: let streamFileURL):
+                upload.validate()
+                upload.responseJSON { response in
+                    self.onSuccess(response)
+                }
+            }
+               })
```
Comment (awazgyawali) [inline ios/Classes/SwiftMultipartRequestPlugin.swift]: Everything looks great and well but can you format this file a bit.

Comment (BEARlogin) [reply ios/Classes/SwiftMultipartRequestPlugin.swift]: Okey, no problem

Comment (BEARlogin) [reply ios/Classes/SwiftMultipartRequestPlugin.swift]: > Everything looks great and well but can you format this file a bit.

Done

**Review by awazgyawali — APPROVED** [review]

**Comment by awazgyawali** [comment]
Thanks for your contribution. I'll upload this to pub shortly.

**Pull request merged**